README.md

```diff
@@ -573,6 +574,16 @@ const proxy = createProxyMiddleware({
 
 Check out [interception recipes](https://github.com/chimurai/http-proxy-middleware/blob/master/recipes/response-interceptor.md#readme) for more examples.
 
+## Node.js 17+: Issues over IPv6 and localhost ([#705](https://github.com/chimurai/http-proxy-middleware/issues/705))
+
+From Node.js 17+: `For dns lookups, Node.js no longer prefers IPv4 over IPV6`.
```

Comment:
@lydell Could you review this updated description?

I'll merge this PR, if this matches your understanding of the new node17+ behaviour.

Thanks in advance!